### PR TITLE
Set downView to private set to give access in unit tests

### PR DESCRIPTION
**Pull request by ezefranca**
## Changes in this pull request

Changed `private var downView: DownView?` to `public private(set) var downView: DownView?` to access it in the unit tests. 

A small start in unit tests can be found in the `ProposalDetailViewControllerTests.swift`

(ps: thanks a lot @diogot for all patience in the review)

(Not totally fixed, but started)
Issue fixed: #104 

### Checklist

- [x] Project builds and runs as expected.
- [x] No new linting violations have been introduced.
- [x] No new bugs have been introduced.
- [x] I have reviewed the [contributing guide](https://github.com/evolution-app/ios/blob/development/.github/CONTRIBUTING.md)
- [ ] I'm attaching to this PR some screenshots (if applicable)

### CODE REVIEW

File: swift-evolution/sourcecode/controllers/proposals/detail/ProposalDetailViewController.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ class ProposalDetailViewController: BaseViewController {
 
     // MARK: - Private properties
     private var proposalMarkdown: String?
-    private var downView: DownView?
+    public private(set) var downView: DownView?
```
Comment (diogot) [inline swift-evolution/sourcecode/controllers/proposals/detail/ProposalDetailViewController.swift]: If this change is done just for tests, maybe `internal private(set)` is more suitable.

File: swift-evolutionTests/controllers/ProposalDetailViewControllerTests.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    func testConformsToWKNavigationDelegate () {
+        XCTAssert(self.proposalDetailViewController.conforms(to: WKNavigationDelegate.self))
+        XCTAssertTrue(proposalDetailViewController.responds(to: #selector(proposalDetailViewController.webView(_:decidePolicyFor:decisionHandler:))))
+    }
+    
+    func testInstanceofDownView() {
+         XCTAssertNotNil(self.proposalDetailViewController.downView)
+    }
+    
+//    func testWKNavigationDelegate () {
```
Comment (diogot) [inline swift-evolutionTests/controllers/ProposalDetailViewControllerTests.swift]: 🤔

File: swift-evolution/sourcecode/controllers/proposals/detail/ProposalDetailViewController.swift (hunk 1 above)
Comment (ezefranca) [reply swift-evolution/sourcecode/controllers/proposals/detail/ProposalDetailViewController.swift]: Fixed!

File: swift-evolutionTests/controllers/ProposalDetailViewControllerTests.swift (hunk 2 above)
Comment (ezefranca) [reply swift-evolutionTests/controllers/ProposalDetailViewControllerTests.swift]: Remove unused code. 🙃

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard [hunk 3]
```diff
@@ -299,11 +299,11 @@ Swift and the Swift logo are trademarks of Apple Inc. For more information about
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="UrU-21-DUf">
-                                <rect key="frame" x="0.0" y="64" width="375" height="603"/>
+                                <rect key="frame" x="0.0" y="0.0" width="375" height="603"/>
```
Comment (diogot) [inline swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: Is this expected/intentional?

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard [hunk 4]
```diff
@@ -299,11 +299,11 @@ Swift and the Swift logo are trademarks of Apple Inc. For more information about
...
                                 <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
                             </view>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="No Selected
Proposal" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="PVc-xW-YFk">
-                                <rect key="frame" x="140.5" y="295" width="94" height="77"/>
+                                <rect key="frame" x="140.5" y="263" width="94" height="77"/>
```
Comment (diogot) [inline swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: Is this expected/intentional?

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard [hunk 5]
```diff
@@ -563,7 +563,7 @@ Swift and the Swift logo are trademarks of Apple Inc. For more information about
     </resources>
     <inferredMetricsTieBreakers>
         <segue reference="rf2-rv-6M0"/>
-        <segue reference="XKc-wh-heR"/>
+        <segue reference="3pf-Ye-P5i"/>
```
Comment (diogot) [inline swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: Is this expected/intentional?

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard (hunk 3 above)
Comment (ezefranca) [reply swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: XCode stuff

Comment (diogot) [reply swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: If you don't mean to change the storyboard maybe would be better to discard those changes.
What do you think?

Comment (ezefranca) [reply swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: removed 👍

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard (hunk 4 above)
Comment (ezefranca) [reply swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: removed 👍

File: swift-evolution/resources/storyboards/Base.lproj/Main.storyboard (hunk 5 above)
Comment (ezefranca) [reply swift-evolution/resources/storyboards/Base.lproj/Main.storyboard]: removed 👍

**Comment by unnamedd** [comment]
As we talked privately, just to keep on records, this is the first step to implement unit tests on EVO. @ezefranca will help us implementing more tests as soon as he can.

Thank you for this start @ezefranca

**Pull request merged**